source/Playnite.DesktopApp/ViewModels/StatisticsViewModel.cs

```diff
@@ -321,6 +322,7 @@ private GameStats FillData(bool filtered)
             ulong hidden = 0;
             ulong favorite = 0;
             ulong totalPlaytime = 0;
+            ulong? totalInstallSize = 0;

```

Comment:
You forgot about this one, doesn't need to be nullable either.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3327/commits/ea0df032562d4678b175973b4e8bfaee7a6cd58a